app/models/form.rb

```diff
@@ -40,7 +40,6 @@ class Form < ActiveRecord::Base
   scope :masters, -> { where(master: true) }
 
   validates :name, presence: true
-  validates :formable_id, uniqueness: { scope: :formable_type, allow_nil: true }
 
   validate :name_is_unique
```

Comment:
some forms are master forms. those show up in a drop down on the petition plugin editor, listed by the name, and you can "apply" the master form, which just duplicates it (including name) and sets the duplicate as the form for that page.